src/hooks/allSitesHooks/useSitePreview.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from "react-query"
+import type { UseQueryResult } from "react-query"
+
+import { SITE_PREVIEW_KEY } from "constants/queryKeys"
+
+import * as AllSitesService from "services/AllSitesService"
+
+import { SitePreviewRequest } from "types/sites"
+
+export const useSitePreview = (
+  userEmail: string,
+  sites: string[]
+): UseQueryResult<SitePreviewRequest[]> => {
+  return useQuery<SitePreviewRequest[]>(
+    [`${SITE_PREVIEW_KEY}`, userEmail],
+    () => AllSitesService.getSitePreview(sites),
+    {
+      retry: false,
+      refetchOnWindowFocus: false,
+      onError: (error) => {
+        console.error(error)
```

Comment:
Hmm do we want to console log here? this would lead to a stream of console logs for us admins. Even if the agency looks at this, it might be unlikely that this would be useful information here

Reply:
okay, the other onError uses `console.log()` too